nixos/tests/grafana/basic.nix

```diff
@@ -1,4 +1,6 @@
-import ./make-test-python.nix ({ lib, pkgs, ... }:
+args@{ pkgs, ... }:
```

Comment:
Why is this needed?

Reply:
I, uhh, actually don't know. I copied the style from Nextcloud test suite. Would love someone to figure that out for me!

Reply:
Just checked that it shouldn't be necessary. I took the liberty to just push the change to your branch.